### DISCUSSION TITLE
#26. VATZ biweekly meeting at 2023-01-18 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#368] [General]
### 1. Overall
<img width="1213" alt="image" src="https://user-images.githubusercontent.com/6308023/210685537-1503fe7e-ae4f-4824-967d-1d6adfecf968.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
24 | 5/21 | 23
25 | 4/19 | 21

---

## In Previous VATZ biweekly meeting

1. Stop command on CLI 
  (ongoing PR https://github.com/dsrvlabs/vatz/pull/387)
2. Monitoring in seperate ...
3. CLI documents
- Add Arguments in documents. 

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/6308023/211111749-77df2467-894a-4146-a2aa-faae6062f152.png">

---

# `VATZ`
<img width="926" alt="image" src="https://user-images.githubusercontent.com/6308023/213172006-2d605668-4674-462d-b030-5cb0a0721b8c.png">

# Official Plugins 
## vatz-plugin-sysutil
<img width="906" alt="image" src="https://user-images.githubusercontent.com/6308023/213179731-c76a13ab-3537-479d-9aca-fa13d7539bb4.png">

## vatz-plugin-cosmoshub
<img width="921" alt="image" src="https://user-images.githubusercontent.com/6308023/213179986-d81b4a7b-9660-47c3-b5aa-881316c0efc2.png">

**Comment by xellos00** [comment]
# 동요

- #378
- #344
- #323
- #338
- #375
- #379
- #376
- #352

**Comment by rootwarp** [comment]
# 준교
- https://github.com/dsrvlabs/vatz/issues/386
  - https://github.com/dsrvlabs/vatz/pull/387
- https://github.com/dsrvlabs/vatz/issues/388

**Comment by skonhwang** [comment]
# 승곤
- no issue

**Comment by heejin-github** [comment]
## 희진
- https://github.com/dsrvlabs/vatz/issues/380
  - https://github.com/dsrvlabs/vatz/pull/381

**Comment by gnongs** [comment]
## 진홍
- no issue

**Comment by kim201212** [comment]
## 형기

- No issue.

**Reply by xellos00** [reply]
@kim201212 
- Admin may requires for Commit & Merge without no review process. 
Please, research once more and ping me.

**Comment by meetrick** [comment]
# 황제
- no issue

**Comment by xellos00** [comment]
- Google Analytic
   - https://github.com/igrigorik/ga-beacon 
- Open Reddit for community. 
- Star
